test/Microsoft.DotNet.Tools.Api/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Text.Unicode;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Tools.Api
+{
+    public class Program
+    {
+        public static string HttpGet(string requestUri)
+        {
+            try
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(requestUri);
+                WebResponse response = request.GetResponseAsync().Result;
+                if (request.HaveResponse)
+                {
+                    using (var reader = new StreamReader(response.GetResponseStream()))
+                    {
+                        return reader.ReadToEnd();
+                    }
+                }
+            }
+            catch { /* choke all errors */ }
+
+            return null;
+        }
+
+        public static int Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Usage: dotnet api <string>");
+                Console.WriteLine("Example: dotnet api xmlReader");
+                Console.WriteLine("Example: dotnet api xmlRe*r");
+                return 0;
+            }
+
+            UrlEncoder urlEncoder = UrlEncoder.Create(UnicodeRanges.BasicLatin);
+            string searchTerm = urlEncoder.Encode(string.Join(" ", args));
+            string requestUri = $"http://packagesearch.azurewebsites.net/Search/?searchTerm={searchTerm}";
```

Comment:
E.g. redirect you to some evil url that downloads gigabytes of data, returns bogus results that crash the parsers, etc. HTTPS should be the default in 2015 :smile: